### DISCUSSION TITLE
10월 31일 주간회의 쓰레드

### DISCUSSION THREAD

**Discussion by her0807** [woowacourse/prolog#1107] [Q&A]
주간회의 내용 공유 바랍니다!

**Comment by euijinkk** [comment]
로드맵 페이지 구현 리뷰 진행중입니다.

https://github.com/woowacourse/prolog/pull/1097

react-admin 으로 로드맵 어드민 만드는 것 구상중입니다

(with @wonsss)

**Comment by al-bur** [comment]
어드민 - 인기있는 학습로그 업데이트 진행중입니다

https://github.com/woowacourse/prolog-admin/issues/11

**Reply by al-bur** [reply]
기존에 구상했던 방법이 불가능한 방법이어서, 현재, 다른 방식으로 구상하여 진행중입니다.

**Comment by hanull** [comment]
어드민 배포하기 진행 중 입니다.
    - [x]  s3 생성
    - [x]  cloud front 생성
    - [x]  버킷 정책 설정
    - [x]  배포 스크립트 작성
    - [ ]  cloud front 캐시 무효화 설정
    - [ ]  prolg-admin에 스크립트 업로드
    - [ ]  배포 테스트

**Comment by JUDONGHYEOK** [comment]
어드민- 멤버관리 진행중 입니다!
https://github.com/woowacourse/prolog-admin/issues/10

**Comment by wishoon** [comment]
로드맵 API 리뷰중입니다~
https://github.com/woowacourse/prolog/pull/1098
